anchor/common/qbft/src/lib.rs

```
@@ -1309,19 +1414,27 @@ where
         }
 
         // Construct unsigned prepare
-        let unsigned_msg =
-            self.new_unsigned_message(QbftMessageType::Prepare, data_hash, vec![], vec![]);
-
-        self.message_sender.send(unsigned_msg);
+        match self.new_unsigned_message(QbftMessageType::Prepare, data_hash, vec![], vec![]) {
+            Ok(unsigned_msg) => {
+                self.message_sender.send(unsigned_msg);
+            }
+            Err(e) => {
+                warn!("Failed to construct prepare message: {:?}", e);
+            }
+        }
     }
 
     // Send a new qbft commit message
     fn send_commit(&mut self, data_hash: D::Hash) {
         // Construct unsigned commit
-        let unsigned_msg =
-            self.new_unsigned_message(QbftMessageType::Commit, data_hash, vec![], vec![]);
-
-        self.message_sender.send(unsigned_msg);
+        match self.new_unsigned_message(QbftMessageType::Commit, data_hash, vec![], vec![]) {
+            Ok(unsigned_msg) => {
+                self.message_sender.send(unsigned_msg);
+            }
+            Err(e) => {
+                warn!("Failed to construct commit message: {:?}", e);
```

Can we silently fail in this function? Doesn't the caller care?